Products/PloneBooking/README.txt

```diff
@@ -2,22 +2,45 @@
 PloneBooking
 ============
 
+See CHANGES.txt as well for complete change log overview for all versions.
 
-Required Products
-=================
-
-* Plone 3.x or Plone 4.x
-
-Installing PloneBooking
+ChangeLog version 3.1.1
```

Comment:
Changelog messages should go into https://github.com/collective/Products.PloneBooking/blob/master/docs/CHANGES.txt